test/irmin-pack/test_flush_reload.ml

```diff
@@ -23,20 +23,13 @@ let suffix_mem repo e =
   try
     match e.k with
     | `c ->
-        Store.S.X.Commit.CA.unsafe_find ~check_integrity:false
-          (snd (Store.S.X.Repo.commit_t repo))
-          k
-        |> Option.is_some
-    | `n ->
-        Store.S.X.Node.CA.unsafe_find ~check_integrity:false
-          (snd (Store.S.X.Repo.node_t repo))
-          k
-        |> Option.is_some
-    | `b ->
-        Store.S.X.Contents.CA.unsafe_find ~check_integrity:false
-          (Store.S.X.Repo.contents_t repo)
-          k
-        |> Option.is_some
+        (* Store.S.X.Commit.CA.unsafe_find ~check_integrity:false *)
+        (*   (snd (Store.S.X.Repo.commit_t repo)) *)
+        (*   k *)
+        (* |> Option.is_some *)
```

Comment:
These comments should be removed (I think?).

Reply:
Absolutely right. Removed them with 4632746b1f9ce94215cadba147ce78180b443630.